### PR TITLE
Added IRIB Nasim link on line 214, 215

### PR DESCRIPTION
**Pull request by mahdijorat**
Its my first time making adding a link, and I have no coding experience. I just followed the guide. Thanks 😊

### CODE REVIEW

File: channels/ir.m3u [hunk 1]
```diff
@@ -211,3 +211,5 @@ http://163.172.138.186/hls/stream.m3u8
 https://voa-lh.akamaihd.net/i/voapnn_7@72817/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="YourTime TV" tvg-language="Persian" tvg-logo="https://yourtime.tv/img/utv-logo.png" group-title="Entertainment",YourTime TV
 https://hls.yourtime.live/hls/stream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="IRIB Nasim" tvg-language="Persian" tvg-logo="https://upload.wikimedia.org/wikipedia/en/a/a0/IRIB_Nasim_Logo.png" group-title="Entertainment",IRIB Nasim
+https://cdn1live.irib.ir/live-channels/smil:nasim/playlist.m3u8?s=xfzwmaVbFGKsE7GxWxNrKw&t=1594967100
```
Comment (freearhey) [inline channels/ir.m3u:215]: Link no longer works. Tested in Elmedia Video Player, VLC and streamtest.in

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check marked link again.

File: channels/ir.m3u (hunk 1 above)
Comment (CrownedKing123) [reply channels/ir.m3u:215]: Hi, I found a working link:
https://live.farsiland.com/nasim/stream.m3u8

**Review by mahdijorat — COMMENTED** [review]
I have found a new working link

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
Great job!

**Pull request merged**